kalliope/core/TriggerLauncher.py

```diff
@@ -20,6 +20,10 @@ def get_trigger(settings, callback):
         :return: The instance of Trigger 
         :rtype: Trigger
         """
+        trigger_folder = None
+        if settings.resources:
+            trigger_folder = settings.resources.trigger_folder
```

Comment:
id settings.resources.trigger_folder always present if settings.resources exists?

Reply:
If not set I guess it triggers an error. We do like this for other launchers.
But you're right maybe we can add a proper error management and log an error to describe the issue.

Reply:
I looked at the [SignalLauncher](https://github.com/kalliope-project/kalliope/blob/a11f4933a5beb1125182dcd9218c14609a60310e/kalliope/core/SignalLauncher.py#L26) how it was done there, and used this method in the TriggerLauncher too. Would it make more sense to use if settings.resources.trigger_folder ?

Reply:
we can leave it like this for now I guess, the folder are defined in the Settings,yml and the error will be raised anyway if missing.